foundation/foundation-mda/src/test/resources/specifications/pipeline-generation.feature

```diff
@@ -47,3 +47,10 @@ Feature: Generating pipeline steps
 		When the profile "data-delivery-pyspark-pipeline" is generated
 		Then a common step parent is created
 		And the common step parent is customizable
+
+	Scenario: PySpark Data Ingest steps in data-flow pipelines can write to a RDMS database
```

Comment:
q: Noticed the file is marked with manual but the steps are pretty flushed out. Are this new test and steps meant to be ran?

Reply:
Good catch - I originally had the tests under the data module feature file and didn't catch the manual tag on the switch. I moved it back and tests have been refactored.